src/webauth/__tests__/webauth.spec.js

```diff
@@ -1,158 +1,73 @@
   const webauth = new WebAuth(auth);
 
-  beforeEach(() => {
-    NativeModules.A0Auth0 = A0Auth0;
-    A0Auth0.reset();
-  });
-
-  describe('clearSession', () => {
-    it('should open log out URL', async () => {
-      await webauth.clearSession();
-
-      const parsedUrl = new URL(A0Auth0.url);
-      expect(parsedUrl.protocol).toEqual('https:');
-      expect(parsedUrl.hostname).toEqual('auth0.com');
-      const urlQuery = parsedUrl.searchParams;
-      expect(urlQuery.get('returnTo')).toEqual(
-        'com.my.app://auth0.com/test-os/com.My.App/callback',
-      );
-      expect(urlQuery.get('client_id')).toEqual('abc123');
-      expect(urlQuery.has('federated')).toEqual(false);
-      expect(urlQuery.has('auth0Client')).toEqual(true);
-    });
-
-    it('should open log out URL with federated=true', async () => {
-      const options = {federated: true};
-      await webauth.clearSession(options);
-
-      const parsedUrl = new URL(A0Auth0.url);
-      expect(parsedUrl.protocol).toEqual('https:');
-      expect(parsedUrl.hostname).toEqual('auth0.com');
-      const urlQuery = parsedUrl.searchParams;
-      expect(urlQuery.get('returnTo')).toEqual(
-        'com.my.app://auth0.com/test-os/com.My.App/callback',
-      );
-      expect(urlQuery.get('client_id')).toEqual('abc123');
-      expect(urlQuery.get('federated')).toEqual('true');
-      expect(urlQuery.has('auth0Client')).toEqual(true);
-    });
-  });
-
-  describe('organizations during log in', () => {
-    it('should build the authorize URL with a organization', async () => {
-      const newTransactionMock = jest
-        .spyOn(webauth.agent, 'newTransaction')
-        .mockImplementation(() =>
-          Promise.resolve({state: 'state', verifier: 'verifier'}),
-        );
-      const showMock = jest
-        .spyOn(webauth.agent, 'show')
-        .mockImplementation(authorizeUrl => ({
-          then: () => Promise.resolve(authorizeUrl),
-        }));
-      const parameters = {organization: 'test-org'};
-      let url = await webauth.authorize(parameters, {});
-
-      const parsedUrl = new URL(url);
-      const urlQuery = parsedUrl.searchParams;
-      expect(urlQuery.get('organization')).toEqual('test-org');
-      newTransactionMock.mockRestore();
-      showMock.mockRestore();
-    });
-
-    it('should build the authorize URL parsing organization from invitation URL', async () => {
-      const newTransactionMock = jest
-        .spyOn(webauth.agent, 'newTransaction')
-        .mockImplementation(() =>
-          Promise.resolve({state: 'state', verifier: 'verifier'}),
-        );
-      const showMock = jest
-        .spyOn(webauth.agent, 'show')
-        .mockImplementation(authorizeUrl => ({
-          then: () => Promise.resolve(authorizeUrl),
-        }));
-      const parameters = {
-        invitationUrl:
-          'https://myapp.com/?invitation=inv123&organization=org123',
+  describe('authorize', () => {
```

Comment:
Should we add a test case with only the required parameters? We already have a test case with all possible parameters.

Reply:
Also, should we add a test case with no parameters, to verify it produces an error?